docker-compose.yml

```diff
@@ -14,6 +14,11 @@ services:
     #      - /dev/gpiomem:/dev/gpiomem
     #        /dev/ttyUSB0:/dev/ttyUSB0
     #        /dev/ttyUSB1:/dev/ttyUSB1
+    #        /dev/ttyACM0:/dev/ttyACM0
```

Comment:
Change these lines to this:
```
    #      - "/dev/ttyUSB0:/dev/ttyUSB0"
    #      - "/dev/ttyUSB1:/dev/ttyUSB1"
    #      - "/dev/ttyACM0:/dev/ttyACM0"
    # If using pi uncomment these lines:
    #      - "/dev/gpiomem:/dev/gpiomem"
    # If using potato uncomment these lines:
    #      - "/dev/gpiochip0:/dev/gpiochip0"
    #      - "/dev/gpiochip1:/dev/gpiochip1"
```
For whatever reason when I tested this docker compose didn't like some of these fields not being enclosed in quotes, even though it worked for others. I don't know why, version difference maybe, but having them all quoted worked fine. The gpio devices are also different depending on which sbc you're using so that needs to be added and documented